launch.go

```diff
@@ -496,6 +496,12 @@ func main() {
 			Value:  "/bin/sh",
 			EnvVar: "SD_SHELL_BIN",
 		},
+		cli.IntFlag{
+			Name:   "build-timeout",
+			Usage:  "Maximum number of seconds to allow a build to run",
+			Value:  DefaultTimeout,
+			EnvVar: "SD_BUILD_TIMEOUT",
```

Comment:
If it is a `flag` or `ENV`, how we are going to pass it in? are we going to change all executor to have a new ENV `SD_BUILD_TIMEOUT `? https://github.com/screwdriver-cd/executor-k8s/blob/master/config/pod.yaml.tim

Reply:
If a flag isn't set and the environment variable isn't set, the default timeout of 90 minutes will be used

Reply:
But we want to support configurable defaultTimeout and also user defined timeout right? The current implementation just make that a bit hard to achieve

Reply:
How do we currently pass in all the variables to the launcher (e.g., build ID, emitter path, etc)?

Reply:
moved it to the issue 👍